src/api/public/apidocs-new/paths/staging_project_name_staging_projects.yaml

```diff
@@ -0,0 +1,94 @@
+get:
+  summary: List all the staging projects of a staging workflow.
+  description: List the staging projects of the staging workflow associated to the specified project.
+  # Authentication is not required
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/staging/staging_projects.yaml'
+          example:
+            staging_project:
+              - name: openSUSE:Factory:Staging:A
+              - name: openSUSE:Factory:Staging:B
+    '404':
+      description: Not Found
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            Project Without Staging Workflow:
+              value:
+                code: project_has_no_staging_workflow
+                summary: "No staging workflow for project 'project_name'"
+            Unknown Project:
+              value:
+                code: unknown_project
```

Comment:
I'm getting `not_found` instead of `unknown_project`

Reply:
changed!